utils/clients/busapps_ingestion_client.py

```diff
@@ -0,0 +1,125 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Client class for ingesting BusApps data via SOAP
+"""
+from suds import Client, WebFault
+from utils.clients.abstract_client import AbstractClient
+from utils.clients.connection_exception import ConnectionException
+from utils.test_settings import BUSAPPS_SETTINGS
+
+
+class BusAppsIngestionClient(AbstractClient):
```

Comment:
What do you think about changing the name to `CycleIngestionClient` and then specify in the class doc string and file doc string that the ingest is via BusinessApps? My only thought is others looking at the code base will have no concept of busapps (e.g. those from other facilities)